backend/src/schema/types/type/Post.gql

```diff
@@ -89,4 +91,10 @@ type Mutation {
     language: String
     categoryIds: [ID]
   ): Post
+  RemovePostEmotions(from: _UserInput!, to: _PostInput!, data: _EMOTEDInput!): Boolean!
+}
+
+type Query {
+  postsEmotionsCountByEmotion(postId: ID!, data: _EMOTEDInput!): Int!
```

Comment:
~I don't like STRANGECase, but I'm not going to request changes because of that - but you and everyone else are welcome to change it in the next PR :wink:~
Found improvements and would like to request changes - so you have the option to implement it in this PR :gift: 

Don't use STRANGECase, pleASE

Reply:
what's STRANGECase? I found this online https://www.artrix.co.uk/whats-on/all-shows/the-strange-case-of-dr-jekyll-and-mr-hyde

the case used in this example is called camel case and is used quite a lot in javascript, do you prefer pascal case?
https://medium.com/better-programming/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841